draft-ietf-quic-multipath.md

```diff
@@ -566,12 +561,13 @@ When only one path is available, servers MUST follow the specifications
 in {{QUIC-TRANSPORT}}.
 
 When more than one path is available, hosts shall monitor the arrival of
-non-probing packets and the acknowledgements for the packets sent over each
-path. Hosts SHOULD stop sending traffic on a path if for at least the period of the
+packets and acknowledgements for packets sent over each
+path. Hosts MAY consider closing a path if for at least the period of the
 idle timeout as specified in {{Section 10.1. of QUIC-TRANSPORT}}
-(a) no non-probing packet was received or (b) no
-non-probing packet sent over this path was acknowledged, but MAY ignore that
-rule if it would disqualify all available paths.
+(a) no packet was received or (b) no packet sent over this path was acknowledged.
+Endpoints that desire to close a path because of the idle timer rule
+MUST do so explicitly by sending a PATH_ABANDON frame on another active path,
+as defined in {{path-close}}.
 
```

Comment:
I think the reference should be {{idle-time-close}}, which is more specific than {{path-close}}.

Reply:
Again, this is the section on idle timeout.

Reply:
Oh, I see. You are moving up the old lines 587-589 to the end of this paragraph. That's fine.